utils/check_task_guides.py

```diff
@@ -72,14 +72,24 @@ def _find_text_in_file(filename, start_prompt, end_prompt):
     "document_question_answering.mdx": transformers_module.models.auto.modeling_auto.MODEL_FOR_DOCUMENT_QUESTION_ANSWERING_MAPPING_NAMES,
 }
 
+# This list contains model types used in some task guides that are not in `CONFIG_MAPPING_NAMES` (therefore not in any
+# `MODEL_MAPPING_NAMES` or any `MODEL_FOR_XXX_MAPPING_NAMES`).
+SPECIAL_TASK_GUIDE_TO_MODEL_TYPES = {
+    "summarization.mdx": ("nllb",),
+    "translation.mdx": ("nllb",),
+}
+
```

Comment:
When cleaning up the entries in mappings with tasks, like removing `nllb` from `MODEL_FOR_SEQ_TO_SEQ_CAUSAL_LM_MAPPING_NAMES`, the check `check_model_list_for_task` fails.

I think the best way is to:
  - still removing `nllb` from XXX__MAPPING_NAMES
  - add this `SPECIAL_TASK_GUIDE_TO_MODEL_TYPES`

WDYT?